src/pages/NewPostPage.tsx

```diff
@@ -0,0 +1,164 @@
+import { Button, Box, CircularProgress, Input, TextField } from "@mui/material";
+import { useState } from "react";
+import { CKEditor } from "@ckeditor/ckeditor5-react";
+import ClassicEditor from "@ckeditor/ckeditor5-build-classic";
+import { AddNewPost } from "../redux/NewPostSlice";
+import { useDispatch, useSelector } from "react-redux";
+import { AppDispatchType, RootState } from "../redux/Store";
+import CustomModal from "../components/CustomModal";
+import { useNavigate } from "react-router-dom";
+import { resetSuccess } from "../redux/NavBarSlice";
+interface newPostDataType {
+  title: string;
+  picture: File | null;
+  description: string;
+}
+export const NewPostPage = () => {
+  const [openCloseModal, setOpenCloseModal] = useState(true);
+  const [inputFieldError, setInputFieldError] = useState({
+    title: false,
+    description: false,
+  });
+  const navigate = useNavigate();
+  const dispatch = useDispatch<AppDispatchType>();
+  const newPostStatus = useSelector((state: RootState) => state.newPostDetail);
+
+  const LoggedInUserData = useSelector((state: RootState) => state.navbarData);
```

Comment:
why loggedInUserData coming from navbarData selector? fix the state names @Rishab-innovative